BackEndApp/tests/utils_tests.py

```diff
@@ -0,0 +1,27 @@
+import sys
+import os
+import mongomock
+
+sys.path.append("..")
+
+from authv1.auth import OTP
+from tests.mocks import MockUser
+
+
+class TestOTP:
+    @mongomock.patch(
+        servers=(
+            (
+                os.getenv("MONGODB_URI"),
+                27017,
+            ),
+        )
+    )
+    def mock_otp(self):
```

Comment:
Can you change the name of this function 'mock_otp' .. as it as calling the otp class

Reply:
Yes, I'll update it ASAP.